PurchasesTests/Mocks/MockSKProduct.swift

```diff
@@ -34,10 +34,7 @@ class MockSKProduct: SKProduct {
 
     @available(iOS 11.2, macCatalyst 13.0, tvOS 11.2, macOS 10.13.2, *)
     override var introductoryPrice: SKProductDiscount? {
-        if #available(iOS 12.2, *) {
-            return mockDiscount
-        }
-        return nil
+        return mockDiscount
```

Comment:
note for other reviewers: the `#available` check wasn't needed, since `SKProductDiscount` is available on 11.2+